server/channels/api4/limits.go

```diff
@@ -29,6 +29,6 @@ func getServerLimits(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := json.NewEncoder(w).Encode(serverLimits); err != nil {
-		c.Logger.Error("Error writing server limits response", mlog.Err(err))
+		c.Logger.Warn("Error writing server limits response", mlog.Err(err))
```

Comment:
Also, I downgrade these log messages while I'm touching the API code. They are now consistent with the rest of the API layer.